.travis.yml

```diff
@@ -1,12 +1,30 @@
 language: php
+
+sudo: false
+
 php:
   - 5.5
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 env:
   global:
     secure: Yc+Xohkr/iEUU7FCQuSLXAE9ywNW9g6CfrM1Ki0Hl+fS15F3AXT7dFY8EyCJ4dP1/oI0dBmwrGWrltXV0XWIjGV1Ms3tefCgQpBBAqwT+hImzVP3RbpZW8Iyo2d0VgiDemQF1LWYD/pKu6d8WljTnv5D77NIMdEJjQ0uzeTLWdw=
-install: composer install --dev
+
+before_install:
+  - composer self-update
+
+install:
+  - composer install --prefer-dist
+
 before_script:
   - git config --global user.email "travis-ci@codeclimate.com"
   - git config --global user.name "Travis CI"
+
+script:
+  - vendor/bin/phpunit
```

Comment:
@jpignata 

Similar to https://github.com/codeclimate/php-test-reporter/pull/33/files#r47942080, and since you're requiring `phpunit/phpunit` in [`composer.json`](https://github.com/codeclimate/php-test-reporter/blob/master/composer.json#L22), you should run the same version if it as installed with `composer`, instead of the one available on Travis.

Otherwise you could possibly see yourself dealing with issues related to tests passing locally, but not on Travis.

This change here explicitly runs `phpunit` as installed with `composer`.